1.32/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM php:7.2-apache
+
+# System Dependencies.
+RUN apt-get update && apt-get install -y \
+		git \
+		imagemagick \
+		libicu-dev \
+		# Required for SyntaxHighlighting
+		python3 \
+	--no-install-recommends && rm -r /var/lib/apt/lists/*
+
+# Install the PHP extensions we need
+RUN docker-php-ext-install mbstring mysqli opcache intl
+
+# Install the default object cache.
+RUN pecl channel-update pecl.php.net \
+	&& pecl install apcu \
+	&& docker-php-ext-enable apcu
+
+# Enable Short URLs
```

Comment:
I just tested this in my own image and it seems to work pretty well.

Reply:
Thanks for testing!